src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Deployment/AzureADDeploymentSource.cs

```diff
@@ -26,8 +26,8 @@ public async Task ProcessDeploymentStepAsync(DeploymentStep step, DeploymentPlan
             var settings = await _azureADService.GetSettingsAsync();
 
             result.Steps.Add(new JObject(
-                new JProperty("name", "AzureAD"),
-                new JProperty("AzureAD", JObject.FromObject(settings))
+                new JProperty("name", "MicrosoftEntraID"),
+                new JProperty("MicrosoftEntraID", JObject.FromObject(settings))
```

Comment:
This is a breaking change, so we shouldn't do it (and it also needs the same rename in the step). However, I'm not sure the original implementation actually worked, because the name here is "AzureAD". while in `AzureADSettingsStep` it's `nameof(AzureADSettings)` (i.e. "AzureADSettings"). Let's continue the latter.

Similarly, the rename in `AzureADDeploymentStep` is a breaking change as well that we should avoid. However, for me it's not even showing up here, please check:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/623cb85e-7c21-4b22-bada-317c88c6c4cd)

Also, the comments on `AzureADDeploymentStep` and `AzureADSettingsStep` are copy-paste leftovers, please fix those too (I know you didn't do them, just housekeeping).

Reply:
I change the names in deployment steps before I submit the commit, I will revert the changes